data/global.lua

```diff
@@ -46,6 +46,13 @@ table.contains = function(array, value)
 	return false
 end
 
+table.merge = function(t1, t2)
```

Comment:
Why keep this function if it is no longer used?

Reply:
I forgot to remove that one but don't you think it'd be useful in the future where someone might need to combine elements of two tables o.0?

If you(& maybe others) still believe that there's no point in keeping this function, I'll remove it.

Reply:
It should be in a separate PR in that case, so as not to create confusion on whether this is related.